### DISCUSSION TITLE
No module named fastapi.__main__; 'fastapi' is a package and cannot be directly executed

### DISCUSSION THREAD

**Discussion by staciax** [fastapi/fastapi-cli#20] [Questions] [answered]
### First Check

- [X] I added a very descriptive title here.
- [X] I used the GitHub search to find a similar question and didn't find it.
- [X] I searched in the documentation/README.
- [X] I already searched in Google "How to do X" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs/README and didn't find an answer.

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```Text
(.venv) backend $ python -m fastapi dev app/main.py
/Users/user/Documents/Github/virgo/backend/.venv/bin/python3: No module named fastapi.__main__; 'fastapi' is a package and cannot be directly executed
```

### Description

i cant run with `python -m fastapi dev app/main.py`

### Operating System

macOS

### Operating System Details

macOS 13.2.1 x86_64

### Python Version

Python 3.12.3

### FastAPI Version

FastAPI CLI version: 0.0.3

**Comment by mattmess1221** [comment, marked as answer]
Use `python -m fastapi_cli dev app/main.py`

**Reply by staciax** [reply]
It works thanks ❤️
